mzLib/UsefulProteomicsDatabases/PtmListLoader.cs

```diff
@@ -63,35 +63,46 @@ public static IEnumerable<Modification> ReadModsFromFile(string ptmListLocation,
         /// <returns></returns>
         public static IEnumerable<Modification> ReadModsFromFile(string ptmListLocation, Dictionary<string, int> formalChargesDictionary, out List<(Modification, string)> filteredModificationsWithWarnings)
```

Comment:
Git shows this as a mess. What happened here was changing the original method parameter from `string filepath` to `Stream`. I then created the method on top which takes in the path, creates the stream, and passes it to the original method